ecommerce/admin.py

```diff
@@ -256,6 +257,44 @@ def get_queryset(self, request):
         return super().get_queryset(request).filter(state=Order.STATE.CANCELED)
 
 
+@admin.register(ReviewOrder)
+class ReviewOrderAdmin(TimestampedModelAdmin):
+    """Admin for ReviewOrder"""
+
+    model = ReviewOrder
+
+    search_fields = [
+        "id",
+        "purchaser__email",
+        "purchaser__username",
+        "reference_number",
+    ]
+    list_display = ["id", "state", "get_purchaser", "total_price_paid"]
+    list_fields = ["state"]
+    list_filter = ["state"]
```

Comment:
Nit-pick but should the filter be changed to something different since all the orders will have a status of review?